src/app/visit/visit.hpp

```diff
@@ -8,12 +8,14 @@
 
 class Doctor;
 class Patient;
+class Room;
 
 class Visit : public std::enable_shared_from_this<Visit>
 {
     std::shared_ptr<Doctor> doctor_;
     std::shared_ptr<Patient> patient_;
     std::vector<Treatment> treatments_;
+    std::shared_ptr<Room> room_;
```

Comment:
At the design part it was mean that it will have an access to whole object. Reason? I see none. Here was the alignment to what we have in our design.